server/api.js

```diff
@@ -697,21 +697,20 @@ router.post("/bid", async (req, res) => {
 			await client.query("BEGIN");
 
 			const checkBidQuery = `
-				SELECT * FROM bid WHERE tender_id = $1 AND bidder_id = $2
```

Comment:
We need to also update the endpoint that allows changing status of a bid. It needs to validate that only one bid is active